docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+# First Steps with DataFrames
+
+The first and imporatant part to work with DataFrames.jl, is to install the `DataFrames.jl`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
```

Comment:
maybe mention in the comment that `]` should be pressed and this is an interactive mode?